python/export_template.py

```diff
@@ -0,0 +1,33 @@
+from urllib import request
+import os
+
+
+root_dir = "discordfs-export"
+headers = {
+    "User-Agent": "discordfs/1.0"
+}
+
+files = [
+
+]
+
+if not os.path.exists(root_dir):
+    os.mkdir(root_dir)
+
+for f in files:
+    path = os.path.join(root_dir, f["channel_name"], f["filename"])
+    if os.path.exists(path):
+        f"Skipping {f['url']} (already downloaded)"
+        continue
+    print(f"Fetching {f['url']}")
+    req = request.Request(f["url"], {}, headers)
+    with request.urlopen(req) as url:
+        if url.status == 429:
+            print("Timeout")
+        b = url.read()
+        if not os.path.exists(os.path.join(root_dir, f["channel_name"])):
+            os.mkdir(os.path.join(root_dir, f["channel_name"]))
+        path = os.path.join(path)
+        with open(path, "wb") as out:
+            out.write(b)
+
```

Comment:
nit: trailing empty line

other than the above, I tested this and it works, at least on a small amount of files.

Reply:
Technically, all files should have trailing empty lines because that's the [POSIX definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206). But I see the rest of this project is lacking empty lines at EOF so I'll still remove it :)